packages/openapi-v3-types/docs.json

```diff
@@ -0,0 +1,4 @@
+{
+  "content": ["index.ts", "src/index.ts", "src/openapi-v3-types.ts"],
```

Comment:
Please update the file names to the actual ones. 
`"src/openapi-v3-spec-types", "src/type-guards"`